tracer/src/Datadog.Trace.SourceGenerators/AspectsDefinitions/AspectsDefinitionsGenerator.cs

```diff
@@ -121,12 +131,67 @@ internal static partial class AspectDefinitions
 
         context.AddSource("AspectsDefinitions.g.cs", SourceText.From(sb.ToString(), Encoding.UTF8));
 
+        if (dest.Tfm.Length > 0 && dest.FolderLocators.Length > 0)
+        {
+            GenerateNative(dest.Tfm[0], dest.FolderLocators[0], aspectClasses, context);
+        }
+
         string FormatLine(string line)
         {
             return $"\"{line.Replace("\"", "\\\"").Replace("RaspIastSink", "Sink")}\",";
         }
     }
 
+    private void GenerateNative(string tfm, in string destFolder, in ImmutableArray<ClassAspects> aspectClasses, SourceProductionContext context)
+    {
+        var tfmName = tfm.Replace(".", "_");
```

Comment:
NIT: It does not really matter since we are not at runtime, but calling Replace(char, char) is more efficient than Replace(string, string) in these cases